packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -246,12 +246,20 @@
 
 // Never show the idle outline when the label is floating, otherwise the label
 // will collide with the idle outline.
-.mdc-text-field__label--float-above ~ .mdc-text-field__idle-outline {
+.mdc-floating-label--float-above ~ .mdc-text-field__idle-outline {
   opacity: 0;
 }
 
 // Show the outline when the label is floating, since the outline has a
 // notch for the label to fit into.
-.mdc-text-field__label--float-above ~ .mdc-text-field__outline {
+.mdc-floating-label--float-above ~ .mdc-text-field__outline {
   opacity: 1;
 }
+
+@include mdc-floating-label-shake-keyframes(box, $mdc-text-field-box-label-position-y);
```

Comment:
The first parameter of all of these keyframes mixins (`$modifier`) should probably be
prefixed with `text-field-` to avoid future collisions with mdc-select.

E.g., `text-field-box`.

Other references to the keyframe names will also need to be updated.